payments-ui-core/res/values-b+es+419/strings.xml

```diff
@@ -78,6 +78,8 @@
   <string name="stripe_continue_button_label">Continuar</string>
   <!-- Label of a button that initiates payment when tapped -->
   <string name="stripe_pay_button_amount">Pagar %s</string>
+  <!-- Card details entry form header title -->
+  <string name="stripe_paymentsheet_add_payment_method_card_information">Información de la tarjeta</string>
```

Comment:
Too late now, we can change next time we touch this, but this should be renamed to remove the "paymentsheet" since it was moved to the shared `payments-ui-core`.